src/app/pages/api/BlogList.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+const BlogList = () => {
+  const [posts, setPosts] = useState([]);
+
+  useEffect(() => {
+    fetch(`${process.env.NEXT_PUBLIC_STRAPI_API_URL}/posts`)
```

Comment:
forma ładowania do ujednolicenia - jeśli gdzieś już jest inaczej to niech wszędzie będzie tak samo